### PR TITLE
[PT][Chapter 01 - 2.mdx] - issue #51

### PR DESCRIPTION
**Pull request by victorescosta**
issue #51

### CODE REVIEW

File: chapters/pt/chapter1/2.mdx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+NLP é um campo da linguística e da Aprendizagem de Máquina (ML) focada em entender tudo relacionado a linguagem humana. O objetivo das tarefas de NLP não é apenas entender palavras soltas individualmente, mas ser capaz de entender o contexto dessas palavras.
+
+A seguir uma lista de tarefas comuns de NLP, com alguns exemplos:
+
+- **Classificação de sentenças completas**: Capturar o sentimento de uma revisão, detectar se um email é spam, determinar se a sentença é gramaticalmente correta ou onde duas sentenças são logicamente relacionadas ou não
+- **Classificação de cada palavra em uma sentença**: Identificar os componentes gramaticais de uma sentença (substantivo, verbo, adjetivo), ou as entidades nomeadas (pessoa, local, organização)
+- **Geração de conteúdo textual**:  Completar um trecho com autogeração textual, preenchendo as lacunas em um texto com palavras mascaradas
+- **Extrair uma resposta de um texto**: Dada uma pergunta e um contexto, extrair a resposta baseada na informação passada no contexto
+- **Generating a new sentence from an input text**: Translating a text into another language, summarizing a text
```
Comment (johnnv1) [inline chapters/pt/chapter1/2.mdx:15]: Faltou esse tópico

File: chapters/pt/chapter1/2.mdx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+NLP não se limita ao texto escrito. Também abarca desafios complexos nos campos de reconhecimento de discurso e visão computacional, tal como a geração de transcrição de uma amostra de áudio ou a descrição de uma imagem.
```
Comment (johnnv1) [inline chapters/pt/chapter1/2.mdx:17]: Também **engloba**...

**Review by johnnv1 — CHANGES_REQUESTED** [review]

**Review by lewtun — COMMENTED** [review]
Thanks for this translation @victorescosta 🚀 !!

Would you like to integrate the suggestions from @johnnv1 and then we can merge?

**Review by victorescosta — COMMENTED** [review]
Thank you, meu caro!

**Review by victorescosta — COMMENTED** [review]
All done!

**Review by lewtun — APPROVED** [review]
Thanks for iterating - this looks great 🔥 !

**Pull request merged**